include/pybind11/cast.h

```diff
@@ -1018,6 +1018,16 @@ template <return_value_policy policy = return_value_policy::automatic_reference,
     return result;
 }
 
+#if PY_MAJOR_VERSION >= 3
```

Comment:
In the documentation I see "New in version 3.3."
Mainly to be slightly more informative, at ~no cost, could you make this `PY_VERSION_HEX >= 0x03030000`?